www_src/pages/project/project.jsx

```diff
@@ -437,6 +468,32 @@ var Project = React.createClass({
     }
   },
 
+  setDestination: function () {
+    var patchedState = this.state.routeData.linkState;
+
+    patchedState = types.link.spec.expand(patchedState);
+
+    // Patch old attributes object to prevent overwritten properties
+    patchedState.attributes.targetPageId = this.state.selectedEl;
+    patchedState.attributes.targetProjectId = this.state.params.project;
+
+    api({
+      method: 'patch',
+      uri: `/users/1/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}/elements/${this.state.routeData.elementID}`,
```

Comment:
This will break when we change links to be able to refer to different projects, do you mind passing along the projectId in the routeData?